src/GraphQL.Tests/DI/DefaultServiceProviderTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.DI
+{
+    public class DefaultServiceProviderTests
+    {
+        private readonly IServiceProvider _serviceProvider = new DefaultServiceProvider();
+
+        [Fact]
+        public void throws_for_null_service_type()
+        {
+            Should.Throw<ArgumentNullException>(() => _serviceProvider.GetService(null));
+        }
+
+        [Fact]
```

Comment:
\+ that one
[Fact]
        public void returns_self_for_defaultserviceprovider()
        {
            _serviceProvider.GetService(typeof(DefaultServiceProvider)).ShouldBe(_serviceProvider);
        }

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/2769